x-pack/plugins/uptime/server/lib/requests/get_monitor_status.ts

```diff
@@ -126,29 +141,59 @@ export const getMonitorStatus: UMElasticsearchQueryFn<
           },
         },
       },
-    };
+    },
+  });
+
+  /**
+   * Perform a logical `and` against the selected location filters.
+   */
+  if (locations.length) {
+    queryParams.body.query.bool.filter.push(getLocationClause(locations));
+  }
 
-    /**
-     * Perform a logical `and` against the selected location filters.
-     */
-    if (locations.length) {
-      esParams.query.bool.filter.push(getLocationClause(locations));
-    }
+  const { body: result } = await uptimeEsClient.search<Ping, typeof queryParams>(queryParams);
+  const afterKeyRes = result?.aggregations?.monitors?.after_key;
+
+  const monitors = result?.aggregations?.monitors?.buckets || [];
+
+  return { afterKeyRes, monitors };
+};
+
+type QueryResponse = PromiseType<ReturnType<typeof executeQueryParams>>;
+
+export const getMonitorStatus: UMElasticsearchQueryFn<
+  GetMonitorStatusParams,
+  GetMonitorStatusResult[]
+> = async ({ uptimeEsClient, filters, locations, numTimes, timespanRange, timestampRange }) => {
+  let afterKey: QueryResponse['afterKeyRes'];
+
+  let monitors: QueryResponse['monitors'] = [];
+
+  do {
+    // today this value is hardcoded. In the future we may support
```

Comment:
I don't think this comment makes sense here anymore. We have had numerous revisions to this function and I think we lost track of the semantic link between the comment and the code it pertains to. I think originally it was intended for [the hardcoded `STATUS` var](https://github.com/elastic/kibana/blob/main/x-pack/plugins/uptime/server/lib/requests/get_monitor_status.ts#L50), but I think we have done away with that so this comment should also go.
```suggestion
```